src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/persistence/PublisherEventsPersistorTest.java

```diff
@@ -78,20 +78,23 @@ public void actionPublished() throws Exception {
         mockActions.add(mockAction3);
         mockActions.add(mockAction4);
 
-        publisherEventsPersistor.persistAction(mockActions);
+        publisherEventsPersistor.persistAction(mockActions, 987654321);
```

Comment:
For the sake of testing sql query correctness, can we also add some out of order actions, i.e. publish another set of actions with an older timestamp after this one. We don't expect the publisher code to do that right now, but since these are *unit* tests, it will validate the sql query against any unexpected future changes.

Reply:
Added UTs where we have entries with earlier timestamp with an increasing ID. Added the UTs to test the SQL query against a field other than the timestamp.